docs/Settings.md

```diff
@@ -2842,6 +2842,16 @@ Launch abort stick deadband in [r/c points], applied after r/c deadband and expo
 
 ---
 
+### nav_fw_launch_vtx_lowpower
+
+Prevents VTX overheating by sending UNARMED to MSP until launch detected.
```

Comment:
Settings.md is generated with a script.

You should update the description in the yaml to this version and regenerate the file by running src/utils/update_cli_docs.py